pkg/kv/dynamodb/store.go

```diff
@@ -165,11 +173,33 @@ func setupKeyValueDatabase(ctx context.Context, svc *dynamodb.DynamoDB, params *
 		},
 	})
 	if err != nil {
-		if _, ok := err.(*dynamodb.ResourceInUseException); !ok {
-			return err
+		if _, ok := err.(*dynamodb.ResourceInUseException); ok {
```

Comment:
if err != nil and err is ResourceInUseException (table already created - as in the case of open twice) we should not wait for table ACTIVE state

Reply:
Which is what I believe I did, no?